modules/nf-core/fastqc/meta.yml

```diff
@@ -11,40 +11,50 @@ tools:
         FastQC gives general quality metrics about your reads.
         It provides information about the quality score distribution
         across your reads, the per base sequence content (%A/C/G/T).
+
         You get information about adapter contamination and other
         overrepresented sequences.
       homepage: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/
       documentation: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/
       licence: ["GPL-2.0-only"]
+      identifier: biotools:fastqc
 input:
-  - meta:
-      type: map
-      description: |
-        Groovy Map containing sample information
-        e.g. [ id:'test', single_end:false ]
-  - reads:
-      type: file
-      description: |
-        List of input FastQ files of size 1 and 2 for single-end and paired-end data,
-        respectively.
+  - - meta:
```

Comment:
Is this `- - ` correct?

Reply:
Yes, the nf-core team is working on these meta.yml files to support tuples. The "old" version presented all the inputs as a flat list in the meta.yml. Now, they have introduced this - - notation to represent composed types (essentially tuples, like tuple(meta, files)).